bin/run-browser-test-env

```diff
@@ -1,84 +1,139 @@
 #!/bin/bash
 
-# DOC: Run the app locally using Docker for use with running browser tests.
+# DOC: Run the app locally using Docker for use with running browser tests. Run with --help to print documentation.
 
-pushd $(git rev-parse --show-toplevel)
+pushd $(git rev-parse --show-toplevel) > /dev/null
 
 set -e
 
-bin/pull-image
-
-if [ -f ".secrets" ]; then
-  . .secrets
-fi
-
-ci="no"
+# Default to continuous integration mode disable.
+continuous_integration=1
+
+# Default to AWS/Localstack emulator.
+cloud_provider="aws"
+emulator="localstack"
+server_url="http://localhost:6645"
+STORAGE_SERVICE_NAME="s3"
+
+#######################################
+# Ensure that only one cloud provider flag was passed tot he script.
+# Exit with error message and status code 1 if more than
+# Globals:
+#   cloud_provider
+#   already_set_cloud_provider
+# Arguments:
+#   1: the cloud provider indicated by the flag currently being processed
+#######################################
+already_set_cloud_provider=1
+ensure_only_one_cloud_provider_flag() {
+  if [[ $already_set_cloud_provider -eq 0 ]]; then
+    local duplicate_providers="${cloud_provider} and ${1}"
+    echo "One cloud provider flag can be set but found both ${duplicate_providers}"
+    exit 1
+  fi
+
+  already_set_cloud_provider=0
+}
 
-# Get cloud provider
-_setArgs(){
+#######################################
+# Process the script's commandline args.
+# Globals:
+#   continuous_integration
+#   already_set_cloud_provider
+#   cloud_provider
+#   emulator
+#   server_url
+#   STORAGE_SERVICE_NAME
+# Arguments:
+#   1: "$@" - full args array for the script
+#######################################
+set_args() {
   while [ "${1:-}" != "" ]; do
     case "$1" in
+      "--help")
+        cat <<-EOF
+Usage: bin/run-browser-test-env [options]
+
+Run a local civiform environment for the browser tests to exercise.
+
+Options:
+
+--ci    Run the environment in CI mode. Removes caching for SBT builds and
+        redirects log statements to a .dockerlogs file.
+
+--aws   Use the AWS emulator (Localstack) for simulating cloud services,
+        cannot be used with --azure flag.
+
+--azure Use the Azure emulator (Azurite) for simluating cloud services,
+        cannot be used with --aws flag.
+EOF
+        exit 0
+        ;;
+
       "--ci")
-        ci="yes"
+        continuous_integration=0
```

Comment:
It looks like you're using 1 to mean don't do CI and 0 to mean to do it which is counter intuitive and backwards of my expectation that 1 is yes/on/do-CI

Reply:
added bin/lib/truth.sh for encapsulating this kinda thing